code/modules/mod/mod_clothes.dm

```diff
@@ -108,16 +96,19 @@
 	desc = "A pair of boots for a MODsuit."
 	icon = 'icons/obj/mod.dmi'
 	icon_state = "boots"
+<<<<<<< HEAD
 	worn_icon = 'icons/mob/mod.dmi'
 	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 100, FIRE = 25, ACID = 25, WOUND = 10)
+=======
+	worn_icon = 'icons/mob/clothing/mod.dmi'
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, FIRE = 0, ACID = 0, WOUND = 0)
+>>>>>>> fd9a7f8a581 (MOD update: Modular Cores (#64042))
```

Comment:
icon path change & armor value change (which is re-changed later BASED on the modsuit)

```suggestion
	worn_icon = 'icons/mob/clothing/mod.dmi'
	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, FIRE = 0, ACID = 0, WOUND = 0)
```